### PR TITLE
feat: [PROD-13567] add placeholder when display of results is disabled

### PR DESCRIPTION
**Pull request by esasova**

### CODE REVIEW

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

File: src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js [hunk 1]
```diff
@@ -119,6 +119,7 @@ export const SimplePowerBIReportEmbed = ({
     () => PowerBIUtils.constructDynamicFilters(dynamicFilters, scenarioDTO),
     [dynamicFilters, scenarioDTO]
   );
+  const resultsDisplayDisabled = reports?.status === 'DISABLED';
```
Comment (nborde-CSM) [inline src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js:122]: I'm not sure about the case of the constant value. As below, we use simple lower cases to set a value. I think full higher cases is reserved for const name itself.

**Review by nborde-CSM — APPROVED** [review]
One remark but LGTM :+1:

File: src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js (hunk 1 above)
Comment (esasova) [reply src/charts/SimplePowerBIReportEmbed/SimplePowerBIReportEmbed.js:122]: as it's a value set in Redux and all store statuses use upper case, like ERROR on line 235, so I choose to do the same for this status :slightly_smiling_face:

**Pull request merged**